api/pkg/apis/v1alpha1/managers/models/models-manager.go

```diff
@@ -84,6 +94,7 @@ func (t *ModelsManager) UpsertSpec(ctx context.Context, name string, spec model.
 	}
 	_, err = t.StateProvider.Upsert(ctx, upsertRequest)
 	if err != nil {
+		log.Debugf(" M (Models): failed to UpsertSpec, name: %s, err: %v, traceId: %s", name, err, span.SpanContext().TraceID().String())
```

Comment:
When we log an actual error this should be Errorf instead of Debugf.

Reply:
Fixed